livedata/src/main/java/io/getstream/chat/android/livedata/converter/ListConverter.kt

```diff
@@ -5,7 +5,6 @@ import com.google.gson.reflect.TypeToken
 import io.getstream.chat.android.client.models.Attachment
 import io.getstream.chat.android.livedata.entity.ReactionEntity
 import io.getstream.chat.android.livedata.gson
-import java.util.*
```

Comment:
Wonder how this wasn't removed by Studio at some point 🤔

Reply:
Because we had that into the project:
https://github.com/GetStream/stream-chat-android-livedata/pull/49/files#diff-1e70daafb475c0ce3fef7d2728279182